src/rabbit_fifo.hrl

```diff
@@ -1,4 +1,6 @@
 
+-type maybe(T) :: undefined | T.
```

Comment:
We [already have](https://github.com/rabbitmq/rabbitmq-common/blob/master/src/rabbit_types.erl#L36) this in `rabbit_types`.

Reply:
`maybe/1` in `rabbit_types` is declared `T | none` which is different